src/layman/layer/rest_layers_test.py

```diff
@@ -47,31 +48,41 @@ def assert_style_file(workspace,
         or all(v is None for v in [layer_style_file, expected_style_file])
 
 
-@pytest.mark.parametrize('source_style_file_path, layer_suffix, expected_style_file_template', [
-    ('sample/style/generic-blue_sld.xml', '_sld', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.sld'),
-    ('sample/style/sld_1_1_0.xml', '_sld11', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.sld'),
-    ('sample/style/funny_qml.xml', '_qml', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.qgis'),
-    ('', '_no_style', None),
+@pytest.mark.parametrize('source_style_file_path, layer_suffix, expected_style_file_template, expected_style_type', [
+    ('sample/style/generic-blue_sld.xml', '_sld', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.sld', 'sld'),
+    ('sample/style/sld_1_1_0.xml', '_sld11', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.sld', 'sld'),
+    ('sample/style/funny_qml.xml', '_qml', '/layman_data_test/users/{workspace}/layers/{layer}/input_style/{layer}.qgis', 'qgis'),
+    ('', '_no_style', None, 'sld'),
 ])
 @pytest.mark.usefixtures('ensure_layman')
 def test_style_correctly_saved(source_style_file_path,
                                layer_suffix,
-                               expected_style_file_template):
+                               expected_style_file_template,
+                               expected_style_type):
     workspace = 'test_style_correctly_saved_workspace'
     layer = 'test_style_correctly_saved_layer' + layer_suffix
     expected_style_file = expected_style_file_template.format(workspace=workspace, layer=layer) if expected_style_file_template else None
     process_client.publish_layer(workspace,
                                  layer,
                                  style_file=source_style_file_path)
     assert_style_file(workspace, layer, expected_style_file)
+    info = process_client.get_layer(workspace, layer)
```

Comment:
Do not call process_client.get_layer, call util.get_publication_info and remove style_type from response